controllers/contacts.js

```diff
@@ -0,0 +1,68 @@
+const contacts = require("../models/contacts.js");
+
+const { HttpError, ctrlWrapper } = require("../helpers");
+
+const Joi = require("joi");
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  phone: Joi.string().required(),
```

Comment:
Schema must be defined in separate file schema.js. You should validate email and phone properly